### PR TITLE
Fix Frame Set Annotations

### PR DESCRIPTION
**Pull request by joshua-dean**
# Fix Frame Set Annotations

## Description
Calling set annotations assumed a `spatial_payload` exists when that's not the case for frame or global annotations.
Added a check for this.

## PR Checklist

- [X] Merged latest main
- [X] Version number in `package.json` has been bumped since last release
- [X] Version numbers match between package `package.json` and `src/version.js`
- [X] Ran `npm install` and `npm run build` AFTER bumping the version number
- [X] Updated documentation if necessary (currently just in `api_spec.md`)
- [X] Added changes to `changelog.md`

## Breaking API Changes
None

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -3516,6 +3516,10 @@ export class ULabel {
         if (subtask == null) {
             subtask = this.state["current_subtask"];
         }
+        // No need to rebuild contianing box for image-level annotation types.
+        if (this.subtasks[subtask]["annotations"]["access"]["spatial_type"] in ["whole-image", "global"]) {
```
Comment (neheller) [inline src/index.js:3520]: Should probably use `NONSPATIAL_MODES` here instead of `["whole-image", "global"]`

**Review by neheller — COMMENTED** [review]
Please also rebuild the `ulabel.js` file with `npm run build`

**Comment by joshua-dean** [comment]
Should have everything sorted out.
Added `.vscode` to `.gitignore` so we don't have to juggle a bunch of build configurations.

**Comment by joshua-dean** [comment]
I was checking through other PRs and realized git can use a [global gitignore](https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer), so I moved my `.vscode` ignore to that instead.

**Review by neheller — APPROVED** [review]
Looks good to me.

**Comment by neheller** [comment]
Note that you might need to merge and bump the version number again

**Pull request merged**